doc/nrf/ug_partition_manager.rst

```diff
@@ -279,3 +279,38 @@ CMake usage
 
       --slot-size $<TARGET_PROPERTY:partition_manager,MCUBOOT_SLOT_SIZE>
 
+
+Predefined configuration
+========================
+Predefined configuration is a feature intended for deployed products which consists of multiple images, but where
+only a subset of the included images will be upgrade-able through firmware update mechanisms.
+In this case, it is important that the images to be upgraded are linked to the same address as what is on the existing product.
+This to avoid accidental overrides, and incorrect configurations.
+Predefined configuration also allow defining new partitions, deleting existing partitions, and re-sizing existing partitions.
+
+Providing predefined configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The predefined configuration is provided through a YAML formatted configuration file.
+This file is similar to the regular :file:`pm.yml` configuration files, except
+that it also defines the start address and size of all partitions.
+The given configuration for a build can be found in :file:`${BUILD_DIR}/zephyr/include/generated/partitions.yml`.
+To apply the current configuration as a predefined configuration, copy this file to the application source directory.
+When the build system sees a file named :file:`partitions.yml` in an applications source directory it is automatically provided to the partition manager script as the predefined configuration.
+
+Removing a predefined partition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A predefined partition is removed by deleting its entry in :file:`partitions.yml`.
+Note that only partitions adjacent with the ``app`` partition can be deleted.
+
+Re-sizing a predefined partition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To re-size a predefined partition, first remove the partition from :file:`partitions.yml` then ensure that the build strategy for that sub image is "Build from source" and update the configured partition size.
+Note that only partitions adjacent with the ``app`` partition can be re-sized.
```

Comment:
This "adjacent" rule is not quite correct since (I think) partitions adjacent to other modified partitions can be modified as well. Can you clarify this?

Reply:
My thought was that in the situation "A B APP C" one will first remove B to get "A APP C" and then A can be modified as well.

It might be useful to define the concept of "upgrade-able area" as you mentioned in one of your earlier comments.